frontend/src/locales/es.json

```diff
@@ -18,6 +18,7 @@
   },
   "back": "Volver",
   "community": {
+    "admins": "Administradores",
```

Comment:
müssten nicht auch hier im Spanischen die Zeilen (30, 31, 32 u 69) analog der englischen und deutschen Version gelöscht werden?

Reply:
Ich habe die nur aus dem Deutschen und Englischen gelöscht weil der linter gemeckert hat. Den Code habe ich ja nicht gelöscht, sondern nur auskommentiert weil wir den ja vielleicht ähnlich wieder woanders nutzen werden.
Die anderen Sprachen werden nicht vom Linter überprüft und so kann ich es da drin lassen für eine schnelle Referenz wenn ich es wieder einbauen will.

Falls ich die anderen Sprachen vom linter überprüfen lassen würde, gäbe es 353 Fehler. Die anderen Übersetzungen außer Deutsch und Englisch scheinen nicht gut gepflegt worden zu sein.

Reply:
Alles klar, das passt für mich. Es war mir nur aufgefallen.